src/v2/ion/i18nTransformer.js

```diff
@@ -256,6 +258,8 @@ const updateLabelForUiSchema = (remediation, uiSchema) => {
 
 };
 
+const isWebAuthnAPIError = ( i18nKey ) => i18nKey.startsWith(WEBAUTHN_API_GENERIC_ERROR_KEY);
```

Comment:
@TianGan-okta - I thought we decided against this implementation, opting to delegate this override to the view layer. Not urgent - but I'm curious to know what the limitation was that caused us to go down this route.